.github/workflows/build-wheel-linux-arm64.yaml

```diff
@@ -0,0 +1,418 @@
+                        sed -i -e 's/LINK_LIBS PUBLIC/LINK_LIBS PUBLIC MLIRDeallocationUtils/g' mlir/mlir-hlo/deallocation/transforms/CMakeLists.txt && \
+                        cmake -S /catalyst/mlir/mlir-hlo -B /catalyst/mhlo-build -G Ninja -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_ASSERTIONS=ON -DMLIR_DIR=/catalyst/llvm-build/lib/cmake/mlir -DPython3_EXECUTABLE=${{ matrix.python_container_path }} -DLLVM_ENABLE_LLD=OFF -DLLVM_ENABLE_ZLIB=OFF -DLLVM_ENABLE_ZSTD=FORCE_ON -DCMAKE_CXX_VISIBILITY_PRESET=hidden && \
+                        cmake --build /catalyst/mhlo-build --target check-mlir-hlo;"                        
+
+    - name: Build Enzyme
+      if: steps.cache-enzyme-build.outputs.cache-hit != 'true'
+      run: |
+        set -x
+        docker run --platform linux/aarch64 \
+            -v /var/run/docker.sock:/var/run/docker.sock \
+            -v `pwd`:/catalyst \
+            -i ${{ matrix.container_img }} \
+            bash -c "   set -x && \
+                        cd /catalyst && \
+                        cat /etc/dnf.conf | sed 's/\[main\]/\[main\]\ntimeout=5/g' > /etc/dnf.conf && \
+                        dnf update -y && \
+                        dnf install -y libzstd-devel gcc-toolset-12 && \
+                        dnf clean all -y && \
+                        source /opt/rh/gcc-toolset-12/enable -y && \
+                        export C_COMPILER=/opt/rh/gcc-toolset-12/root/usr/bin/gcc && \
+                        export CXX_COMPILER=/opt/rh/gcc-toolset-12/root/usr/bin/g++ && \
+                        rm -rf ${{ matrix.python_container_path }} && \
+                        ln -s /opt/_internal/cpython-${{ matrix.python_version }}.${{ matrix.python_sub_version }}/bin/python3 ${{ matrix.python_container_path }} && \
+                        export PYTHON=${{ matrix.python_container_path }} && \
+                        export PATH=/catalyst/llvm-build/bin:/opt/_internal/cpython-${{ matrix.python_version }}.${{ matrix.python_sub_version }}/bin:/opt/rh/gcc-toolset-12/root/usr/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin && \
+                        ${{ matrix.python_container_path }} -m pip install numpy pybind11 PyYAML cmake ninja && \
+                        cmake -S /catalyst/mlir/Enzyme/enzyme -B /catalyst/enzyme-build -G Ninja -DCMAKE_BUILD_TYPE=Release -DLLVM_DIR=/catalyst/llvm-build/lib/cmake/llvm -DENZYME_STATIC_LIB=ON -DCMAKE_CXX_VISIBILITY_PRESET=hidden && \
+                        cmake --build /catalyst/enzyme-build --target EnzymeStatic-18;"
+
+  catalyst-macos-wheels-arm64:
+    needs: [constants, build-dependencies]
+    strategy:
+      fail-fast: false
+      matrix:
+        python_version: [3.9]
+        python_sub_version: [19]
+        python_container_path: ["/usr/bin/python3"]
+        numpy_include_path: ["/opt/_internal/cpython-3.9.19/lib/python3.9/site-packages/numpy/core/include"]
```

Comment:
Why do we expect this `_internal` path to exist at the moment this scenario gets executed?

Reply:
This is a predefined path within manylinux containers to house the different versions of Python.

Reply:
Done